### ISSUE TITLE
GTK+ tooltip can be used unintentionally

### ISSUE THREAD

**Issue by iRi-E** [pitkali/pos-tip#15]
The master branch in this repository has been forked in an ancient bzr commit upstream, so [an important fix that forces not to use GTK+ tooltip](https://github.com/iRi-E/pos-tip-el/commit/4b35a615c8b3a62be3b3e3965a16e6e72f34f7b6) is not included.

You should cherry-pick the commit from my (upstream) git repo.
```
git remote add upstream https://github.com/iRi-E/pos-tip-el.git
git fetch upstream
git cherry-pick 4b35a615c8b3a62be3b3e3965a16e6e72f34f7b6
```

**Comment by pitkali** [comment]
Whoa — that does look like a proper upstream repo that I failed to find all these years ago when I fixed my issue and tried to figure out where to submit the patch. I can see that you already applied my fixes, so I'll skip over the suggestion that I submit them to you instead.

Should we talk to the MELPA people to switch to your repo, then? It seems somewhat more complete, and I haven't even been using Emacs much lately.
